README.md

```diff
@@ -92,6 +95,40 @@ If you want to:
 Then run the following command: 
 `python job-cli.py -prod --auth auth-credentials.base64 --directory /opt/foo`
 
+### Until Parameter
+
+If you only want to pull claims data added to the CMS system before a certain date use the `--until` parameter.
+The expected format follows the typical
+ISO date time format of `yyyy-MM-dd'T'HH:mm:ss.SSSXXX+/-ZZ:ZZ`.
+
+For requests using FHIR R4, a default `_until` value is supplied if one is not provided. The value of the default `_until`
+parameter is set to the current date and time.
+
+Examples:
+1. March 1, 2024 at 3 PM EST -> `2024-03-01T15:00:00.000-05:00`
+2. May 31, 2024 at 4 AM PST -> `2024-05-31T04:00:00-08:00`
+
+### Files
+
+1. /directory/jobId.txt -- id of the job created
+2. /directory/response.json -- list of files created
+3. /directory/*.ndjson -- downloaded results of exports
+
+### Limitations
+
+1. Assumes all scripts use the same directory
+2. Assumes all scripts use the same base64 encoded AUTH token
+
+### Example
```

Comment:
Same comment as the other ReadMe to reformat/reword slightly. 

### Example scenario 

1. Start a job running against production.
2. Use base64 encoded credentials stored in `auth-credentials.base64.`
3. Save all results for this job to the directory /opt/foo.
4. Filter for claims data updated before March 1, 2020 [insert time here] by running the following command: